ansible/templates/usr/local/bin/backup-to-s3.sh.j2

```diff
@@ -29,41 +29,40 @@ if [[ -z ${1:-} ]]; then
 fi
 
 SOURCE="${1}"
-FILE="${SOURCE}-$(date +'%Y%m%d-%H%M%S')"
-HASH="sha256" # Choose from md5/sha1/sha224/sha256/sha384/sha512
+FILE_TITLE="${SOURCE}-$(date +'%Y%m%d-%H%M%S')"
```

Comment:
`date -Iminutes` would be a bit nicer, as it's ISO8601, but I can understand this would be a hassle to change (and we don't really care about timezones here)

Reply:
Nit: `FILE_NAME` would be better than `FILE_TITLE`

Reply:
> `date -Iminutes` would be a bit nicer, as it's ISO8601, but I can understand this would be a hassle to change (and we don't really care about timezones here)

I think I did this so we won't end up with colons in the filename, which aren't friendly for Windows people trying to download a backup. (And as you said, we don't really need timezones.)

> Nit: FILE_NAME would be better than FILE_TITLE

This was done on purpose, because this is the filename without the extension. Later on I used $FILE_NAME as the var where the extension is appended to $FILE_TITLE.

Reply:
Ah, alright! Maybe FILE_PREFIX is better in that case? (I assumed it was the final filename)

Reply:
`<bikeshedding>`Well it is the entire "body" of the filename, so prefix sounds a bit weird to me. Since it's only set once, I'll add a comment there.`</bikeshedding>`